crates/revive-types/Cargo.toml

```diff
@@ -0,0 +1,40 @@
+[package]
+name = "ink_revive_types"
+version.workspace = true
+authors.workspace = true
+edition.workspace = true
+license.workspace = true
+description = "ink! revive primitives (placeholder)"
```

Comment:
```suggestion
description = "Primitives copied from Parity's Polkadot SDK module `pallet-revive` . Copied to reduce the build times of ink! contracts."
```

Reply:
My bad, fixed in https://github.com/use-ink/ink/pull/2657/commits/6db9d5f9c19cecbcdeee7aa4d9c4185e91f3fe52